keps/sig-network/20190125-ingress-api-group.md

```diff
@@ -161,146 +159,157 @@ consistent with the syntax supported by any of the common proxy vendors:
 
 Among cloud providers, there is also inconsistent levels of support
 for regular expression-based path matching. See the load-balancer
-documentation for [aws][aws-re], [gcp][google-re], [azure][azure-re].
+documentation for [AWS][aws-re], [GCP][google-re], [Azure][azure-re].
```

Comment:
https://github.com/zalando/skipper use Go regexp engine which is RE2

Reply:
Adding link.